application/src/test/java/org/opentripplanner/transit/service/DefaultTransitServiceTest.java

```diff
@@ -107,43 +127,57 @@ static void setup() {
     var siteRepository = TEST_MODEL.siteRepositoryBuilder()
       .withRegularStop(STOP_A)
       .withRegularStop(STOP_B)
+      .withRegularStop(STOP_C)
+      .withRegularStop(STOP_ONE)
       .withStation(STATION)
       .build();
 
     var deduplicator = new Deduplicator();
-    var transitModel = new TimetableRepository(siteRepository, deduplicator);
+    var timetableRepository = new TimetableRepository(siteRepository, new Deduplicator());
     var canceledStopTimes = TEST_MODEL.stopTimesEvery5Minutes(3, TRIP, "11:30");
     var canceledTripTimes = TripTimesFactory.tripTimes(TRIP, canceledStopTimes, deduplicator)
       .createRealTimeFromScheduledTimes()
       .cancelTrip()
       .build();
-    transitModel.addTripPattern(RAIL_PATTERN.getId(), RAIL_PATTERN);
+    timetableRepository.addTripPattern(RAIL_PATTERN.getId(), RAIL_PATTERN);
 
     // Crate a calendar (needed for testing cancelled trips)
     CalendarServiceData calendarServiceData = new CalendarServiceData();
     var firstDate = LocalDate.of(2024, 8, 8);
     var secondDate = LocalDate.of(2024, 8, 9);
+    var today = LocalDate.now();
```

Comment:
Now that we pass in the date it should be possible to always use fixed dates in the test, right?